repository/mock/release.go

```diff
@@ -18,7 +18,7 @@ func (m *ReleaseRepository) Create(ctx context.Context, rls svcmodel.Release) er
 	return args.Error(0)
 }
 
-func (m *ReleaseRepository) ReadForProject(ctx context.Context, projectID, releaseID uuid.UUID) (svcmodel.Release, error) {
+func (m *ReleaseRepository) Read(ctx context.Context, projectID, releaseID uuid.UUID) (svcmodel.Release, error) {
```

Comment:
Naming will be updated once the conclusion is reached here - https://github.com/jan-zabloudil/release-manager/pull/69#pullrequestreview-2059661739